source/globalCommands.py

```diff
@@ -1703,35 +1703,92 @@ def script_reportClipboardText(self,gesture):
 	script_reportClipboardText.category=SCRCAT_SYSTEM
 
 	def script_review_markStartForCopy(self, gesture):
-		self._copyStartMarker = api.getReviewPosition().copy()
+		reviewPos = api.getReviewPosition()
+		startMark = reviewPos.copy()
+		reviewPos.obj._copyStartMarker = startMark # represents the start location

```

Comment:
I think it might be worth adding a comment here explaining why we put this on obj; i.e. because this way, it gets cleaned up when obj gets cleaned up.